fluent-bit/http/values.yaml

```diff
@@ -174,24 +174,26 @@ fluent-bit:
           Header                private_key ${PRIVATE_KEY}
           compress              gzip
           Retry_Limit           False
+          net.keepalive         off
 
       @INCLUDE output-systemd.conf
     
     extraFiles:
-      input-systemd.conf: |-    
+      input-systemd.conf: |-
         [INPUT]
           Name systemd
           Tag host.*
           Systemd_Filter _SYSTEMD_UNIT=kubelet.service
-          Read_From_Tail On 
-          Mem_Buf_Limit 5MB  
+          Read_From_Tail On
+          Mem_Buf_Limit 5MB
+          DB /var/log/host-journal.db
```

Comment:
why do we need it ? not sure its really necessary for systemd logs

Reply:
i guess its not if we are using Read_From_Tail (read only new logs coming to journal). but if we remove that setting then even if the pod die, no systemd logs will be dropped.

Reply:
reverted this change